roombasim/pittras/ai/land_demo_controller.py

```diff
@@ -0,0 +1,12 @@
+
+from roombasim.ai import Controller
+
+class LandDemoController(Controller):
+
+    def setup(self):
+        # TODO(mbilker): initialize the drone to an arbitrary height above
+        # PITTRAS_LAND_HEIGHT_TOLERANCE
+
+        self.task_controller.switch_task(
+            'LandTask'
```

Comment:
make sure to export the LandTask in `roombasim/pittras/tasks/__init__.py` and add it to the `roombasim.pittras.config` file